File: 1344-angle-between-hands-of-a-clock/1344-angle-between-hands-of-a-clock.py
class Solution:
    def angleClock(self, hour: int, minutes: int) -> float:
        if hour == 12:
            hour = 0

        hour_position = (hour * 5) + (minutes / 60) * 5

        return  min(abs(minutes - hour_position), 60 - minutes + hour_position, 60 - hour_position + minutes) * 6

File: 1344-angle-between-hands-of-a-clock/1344-angle-between-hands-of-a-clock.java
class Solution {
    public double angleClock(int hour, int minutes) {
        if (hour == 12) hour = 0;

        double hrPosition = (hour * 5) + (minutes / 60.0) * 5.0;

        return Math.min(Math.abs(minutes - hrPosition), Math.min(60 - minutes + hrPosition, 60 - hrPosition + minutes)) * 6;
        
    }
}